File: hari/polindrome.c
#include <stdio.h>
void  main()
{
    int a,b,c,revers_b;
    printf ("enter the number:\n");
    scanf("%d",&a);
 while(a>10)  
 {  
     b=a%10;
    revers_b=revers_b*10+b;
    a/=10;
     
 }
    
    if(a==revers_b)
    {
        printf("polidrome");}
       
        else
       { 
        printf("not polindrome");
       }
}

File: python/polindrome.py
# polindrome
def me(i):
	if i%2==0:
		print("not prime number")
	else:
		print("prime number") 
def m():
	mystr=input("enter the string :")
	mystr=mystr.casefold()
	reverstr=reversed(mystr)
	if list(mystr)==list(reverstr):
		print("polindrome")
		print(list(mystr))
		me(mystr)
	else:
		print ("not polindrome")
		print (list(reverstr))
		me(mystr)
m()


